src/Microsoft.ML.Data/Scorers/PredictionTransformer.cs

```diff
@@ -55,7 +53,8 @@ public abstract class PredictionTransformerBase<TModel, TScorer> : IPredictionTr
 
         public bool IsRowToRowMapper => true;
 
```

Comment:
Could you add a line of XML?

Reply:
I suppose I could.

---
In reply to: [254558375](https://github.com/dotnet/machinelearning/pull/2453#discussion_r254558375) [](ancestors = 254558375)